### PR TITLE
Docs/faqs

### PR DESCRIPTION
**Pull request by 0x-r4bbit**

### CODE REVIEW

File: src/guide/configuring-labs.md [hunk 1]
```diff
@@ -107,11 +109,26 @@ $ python main.py --foo=bar some-value
 
 A very good example to see script parameters in action is MachineLabs [Neural Style Transfer Lab](https://machinelabs.ai/editor/rJQrQ5wjZ/1506415557004-HkTTQ5Dob?file=ml.yaml&tab=editor). 
 
+## Hardware configuration
+
+Obviously, the hardware on which we execute our experiments has a big impact on how much time is being spent on for example training a neural net. That's why we often want to make sure our code is executed on **GPU-accelerated machines**. MachineLabs supports CPU and GPU hardware.
+
+We will open up possible hardware configurations in the future but for now every lab is executed on CPU hardware by default. To run our labs on GPU-accelerated machines, all we have to do is to set the `hardwareType` configuration to `gpu` like this:
```
Comment (cburgdorf) [inline src/guide/configuring-labs.md]: We may want to add here that besides setting the `hardwareType` the user also has to pick a GPU compatible docker image

File: src/guide/faqs.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: FAQs
+type: guide
+order: 1
+---
+
+<p class="tip">MachineLabs is currently in private beta. <strong>To follow this guide you need to be part of the private beta program</strong>. If you haven't done so yet, login at <a href="https://machinelabs.ai/editor" title="MachineLabs Editor">machinelabs.ai</a> to become part of the next batch. We onboard new users weekly.</p>
```
Comment (cburgdorf) [inline src/guide/faqs.md:7]: Can we make this an include so that we keep things more DRY

File: src/guide/faqs.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### What libraries are supported?
+
+Right now we provide docker images with languages and libraries listed [here](/guide/configuring-labs.html#Lab-environment). However, we keep adding more docker images in the future to add support for more libraries in different versions. 
+
+In fact, soon you'll be able to spin up your own docker images with any environment you like!
+
+### How much execution time do I have?
+
+During our private beta you get **72 hours** of CPU time.
```
Comment (cburgdorf) [inline src/guide/faqs.md]: *per month*

File: src/guide/faqs.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+You have unlimited internet access in your lab, which means you can access any dataset you like. The easiest way to fetch datasets would be to specify them as inputs in the ml.yaml file. Checkout our docs on [Lab Inputs](/guide/configuring-labs.html#Lab-inputs-datasets) to learn how to do that.
+
+We're also working on a new feature that lets you mount any data of any size at runtime of your lab, so you don't have to download your inputs every time you hit the run button.
+
+### How can I save and download output data?
+
+All you have to do is to write your generated files to `./outputs` and we'll make sure to upload them once your execution is done, so they become available in the outputs view. Take a look at [this lab](https://machinelabs.ai/editor/rJQrQ5wjZ/1506415557004-HkTTQ5Dob?file=ml.yaml&tab=outputs) to see it in action.
+
+Another way to access any generated data is via our [REST API](https://blog.machinelabs.ai/2017/10/16/new-rest-api-and-folder-support/).
```
Comment (cburgdorf) [inline src/guide/faqs.md]: How about 

>Outputs can also be requested via our [REST API](https://blog.machinelabs.ai/2017/10/16/new-rest-api-and-folder-support/)

File: src/guide/faqs.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+If you think the latter applies, please reach out to us [via email](mailto:hello@machinelabs.ai) so we can enable GPU support for you right away!
+
+### Why do I have to pay for GPU support?
+
+GPU is very expensive. We are a small team with a tight budget trying to move the whole Machine Learning community forward. We believe it is important to not leave AI exclusively to the bigger companies but rather have some small indie teams go working on it as well. You can make a difference by supporting us on our mission!
+
+### How much do I need to pledge?
+
+We  enable GPU support for all backers that are backing $10 or more per month.
```
Comment (cburgdorf) [inline src/guide/faqs.md]: Is there an extra space between _We_ and _enable_?

File: src/guide/rest-api.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+title: REST API
+type: guide
+order: 8
+---
+
+MachineLabs comes with a REST API that not only let's us fetch metadata about our labs and executions, but also generated output like **trained models** from other labs, so we can build on top of those.
```
Comment (cburgdorf) [inline src/guide/rest-api.md]: >let's us fetch metadata about our labs and executions

We don't have such APIs yet

**Comment by cburgdorf** [comment]
Great work! Left some comments.

File: src/guide/configuring-labs.md (hunk 1 above)
Comment (0x-r4bbit) [reply src/guide/configuring-labs.md]: Good catch

File: src/guide/faqs.md (hunk 2 above)
Comment (0x-r4bbit) [reply src/guide/faqs.md:7]: I need to figure out how that is done in hexo

File: src/guide/faqs.md (hunk 3 above)
Comment (0x-r4bbit) [reply src/guide/faqs.md]: added

File: src/guide/faqs.md (hunk 4 above)
Comment (0x-r4bbit) [reply src/guide/faqs.md]: Done

File: src/guide/faqs.md (hunk 5 above)
Comment (0x-r4bbit) [reply src/guide/faqs.md]: yupp, fixed

File: src/guide/rest-api.md (hunk 6 above)
Comment (0x-r4bbit) [reply src/guide/rest-api.md]: Okay, rolled back

File: src/guide/faqs.md (hunk 2 above)
Comment (0x-r4bbit) [reply src/guide/faqs.md:7]: Okay I've found https://www.npmjs.com/package/hexo-include

It says the include path is relative to the source path. I tried all possible variations now but I can't make it find the file to include right now.

Will revisit.

**Pull request merged**